flow_screen_components/richTextInputFSC/README.md

```diff
@@ -1,3 +1,45 @@
 # Rich Text Edit Control #
 
 https://unofficialsf.com/rich-text-input-flow-screen-component/
+
+
+# inputRichTextFSC
+Rich Text enhancements for Flows
+
+### Summary:
+IMPORTANT:  Use richerText = true input attribute if you want to leverage these enhanced features.  Otherwise leave empty and regular input Rich Text component will be used.
+1)	Block disallowed words and symbols
+    a.	Control whether this is a block that prevents next/finish of flow.
+2)	Search and Replace custom text
+3)	Auto-Replace suggested words
+4)	Revert mass changes from search/replace or suggested words
+
+### Admin Instructions:
+1)  Follow Input/Output map below.
+2)  Hard Block will prevent user from advancing the Flow screen if there are disallowedWords or disallowedSymbols used.
+3)  Default for disallowed Words/Symbols if hardBlock is false will be a Warning message only.  Setting hardBlock to true will set them to Error messages.
+
+|Parameter	               |I	   |O	     |Information 
+|--------------------------|-----|-------|-------------------------------------------------------------------------------|
+|**richerText**            |X    |       |  Boolean.  Set to true if you want to use enhanced rich text.  Default if false (regular input component)   |
+|**autoReplaceMap**	           |X	   |       |JSON formatted key:value map.  (example => {"Test": "GreatTest™"} )            |
+|**disallowedSymbols**	       |X	   |       |Comma-separated list of words to block.  Example: /,@,*                        |
+|**disallowedWords**	         |X    |	     |Comma-separated list of words to block.  Example: bad,worse,worst              |
+|**warnOnly**	               |X	   |	     |Boolean.  Set to True if you want to allow Next even where disallowed Symbol or Word remains.  Default is false.   |
+|**richText**	                 |X	   |X	     |Input and output RichText that you’ll be editing                               |
+|**label**                      |X     |X        |Input to provide field-level label if desired                                 |
+
+
+### User Instructions:
```

Comment:
How do we get these in front of users? Perhaps add an infobubble or question mark button on the toolbar that pops up this information? You can boil this down to a much smaller set of info.

Reply:
Added lightning-helptext next to button icons in latest version.  Did a simple blurb.  We could allow for an input attribute as well, but hard-coding for now as replacing the read-me for user facing content.